.github/workflows/release.yml

```diff
@@ -0,0 +1,108 @@
+name: DiAGRAM release to live
+on:
+  workflow_dispatch:
+permissions:
+  id-token: write
+run-name: DiAGRAM e2e Tests - dev
+jobs:
+  create-environment:
+    runs-on: ubuntu-latest
+    environment: dev
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/checkout@v4
+        with:
+          repository: 'nationalarchives/DiAGRAM-terraform'
+          path: terraform
+      - uses: hashicorp/setup-terraform@v3
+        with:
+          terraform_version: 1.12.2
+      - uses: aws-actions/configure-aws-credentials@v4.1.0
+        with:
+          role-to-assume: arn:aws:iam::${{ secrets.AWS_ACCOUNT_NUMBER }}:role/DiagramTerraformRole
+          aws-region: eu-west-2
+          role-session-name: TerraformRole
+      - run: |
+          terraform init
+          terraform workspace select dev
+          terraform apply --target module.site -auto-approve
+        working-directory: terraform
+      - run: |
+          npm ci
+          npm run build
+        working-directory: app
+      - run: aws s3 sync ./app/dist s3://nata-dia3-dev-site/ --delete
+  cypress-e2e:
```

Comment:
Should we prefix it with a verb to match the other steps, like "run-cypress-e2e-tests"